File: src/Entity/VeloAl.php
<?php

namespace App\Entity;

use App\Repository\VeloAlRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=VeloAlRepository::class)
 */
class VeloAl
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $marque;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $type;

    /**
     * @ORM\Column(type="float")
     * @Assert\Regex(
     *     pattern="/[0-9]{3}/"
     *  )
     */
    private $prix;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Regex(
     *     pattern="/[0-9]{2}/"
     *  )
     */
    private $quantite;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getMarque(): ?string
    {
        return $this->marque;
    }

    public function setMarque(string $marque): self
    {
        $this->marque = $marque;

        return $this;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(string $type): self
    {
        $this->type = $type;

        return $this;
    }

    public function getPrix(): ?float
    {
        return $this->prix;
    }

    public function setPrix(float $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    public function getQuantite(): ?int
    {
        return $this->quantite;
    }

    public function setQuantite(int $quantite): self
    {
        $this->quantite = $quantite;

        return $this;
    }
}

File: E-bike_java/src/entities/Velo_al.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import javafx.scene.image.ImageView;

/**
 *
 * @author amine
 */
public class Velo_al {
    
    
    private int id;
    private String marque;
    private String Type;
    private int prix;
    private int quantite;
    private String image;
    private ImageView img;

    public ImageView getImg() {
        return img;
    }

    public void setImg(ImageView img) {
        this.img = img;
    }

    

    public Velo_al(int id, String marque, String Type, int prix, int quantite, String image) {
        this.id = id;
        this.marque = marque;
        this.Type = Type;
        this.prix = prix;
        this.quantite = quantite;
        this.image = image;
    }

    public Velo_al() {
    }

    public Velo_al(String marque, String Type, int prix, int quantite, String image) {
        this.marque = marque;
        this.Type = Type;
        this.prix = prix;
        this.quantite = quantite;
        this.image = image;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getMarque() {
        return marque;
    }

    public void setMarque(String marque) {
        this.marque = marque;
    }

    public String getType() {
        return Type;
    }

    public void setType(String Type) {
        this.Type = Type;
    }

    public int getPrix() {
        return prix;
    }

    public void setPrix(int prix) {
        this.prix = prix;
    }

    public int getQuantite() {
        return quantite;
    }

    public void setQuantite(int quantite) {
        this.quantite = quantite;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
    
    
}
